x-pack/plugins/infra/public/components/alerting/logs/expression_editor/editor.tsx

```diff
@@ -4,35 +4,45 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React, { useCallback, useMemo, useEffect, useState } from 'react';
-import { EuiButtonEmpty } from '@elastic/eui';
+import React, { useCallback, useMemo, useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiButtonEmpty, EuiLoadingSpinner, EuiSpacer, EuiButton, EuiCallOut } from '@elastic/eui';
+import { useMount } from 'react-use';
 import { FormattedMessage } from '@kbn/i18n/react';
 import {
   ForLastExpression,
   // eslint-disable-next-line @kbn/eslint/no-restricted-paths
 } from '../../../../../../triggers_actions_ui/public/common';
 // eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { IErrorObject } from '../../../../../../triggers_actions_ui/public/types';
-import { useSource } from '../../../../containers/source';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
```

Comment:
Do you know why this path might be restricted? Maybe it's a bug and it shouldn't

Reply:
Yeah, I need to follow this up with the Alerting team. There's a few types we're (logs and metrics) using that aren't exported from `public` or `server`, and so aren't really supposed to be used. Once they're bumped up to being part of the `public` or `server` contract, I'll remove the lint overrides. I'll handle in a followup 👍